src/algorithms/pid/ParticlesWithPID.cc

```diff
@@ -92,6 +92,12 @@ namespace eicrecon {
                                    deta < m_cfg.etaTolerance &&
                                    dsphi < sinPhiOver2Tolerance;
 
+                // Matching kinematics with the static variables doesn't work at low angles and within beam divergence
+                // TODO - Maybe reconsider variables used or divide into regions
+                if ((p_eta < -5) && (edm4eic::eta(mom) < -5)) {
```

Comment:
Should this equally treat far backward and far forward?

Reply:
Probably, I have added this now.

Reply:
@ajentsch @rahmans1 fyi; I think this only affects the B0 Acts tracks, right?